tests/testPsfFitter.py

```diff
@@ -123,7 +123,9 @@ def testFixedModel(self):
         ellipses3 = model.makeEllipseVector()
         for i in range(len(ellipses2)):
             self.assertFloatsAlmostEqual(ellipses1[i].getParameterVector(), ellipses2[i].getParameterVector())
-            ellipses3[i] = msf.getComponents()[i].getEllipse()  # need to convert ellipse parametrization
+            # need to convert ellipse parametrization
+            ellipses3[i].setCore(msf.getComponents()[i].getEllipse().getCore())
+            ellipses3[i].setCenter(msf.getComponents()[i].getEllipse().getCenter())
```

Comment:
You guessed it.  The old code relied on invoking a C++ assignment operator that converted from the source ellipse parameterization to the destination one.  That used to work okay through the Swig-wrapped `std::vector`s, but it doesn't work when assigning to a `list`.

Clearly, this pattern is quite fragile in Python, and we should try to move away from it.  This comment was a helpful reminder to create an issue for that: [DM-9592](https://jira.lsstcorp.org/browse/DM-9592).